### PR TITLE
Added tts inference page

### PR DESCRIPTION
**Pull request by Nanayeb34**
# Description

Added  TTS inference script that enables you to make API calls to the TTS model

# Testing

Tested in postman
image

# Reviewers

@wsedor @kojomensahonums

### CODE REVIEW

File: text-to-speech/TTS_inference_API_notebook .ipynb [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+      "metadata": {
+        "id": "xbHL2ZCNNYu3"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!ngrok config add-authtoken #place your access token here"
```
Comment (wsedor) [inline text-to-speech/TTS_inference_API_notebook .ipynb:139]: please add a comment above this line to give some guidance like https://github.com/nsmq-ai/nsmqai/pull/51/files#diff-d755e2bb4a53a4cf5f1dcfb59c460f3cbc400da5cdb40a512d867038c1efc11cR194

**Comment by wsedor** [comment]
@Nanayeb34 please fill out the PR details section. it is currently using the template

**Comment by Nanayeb34** [comment]
> @Nanayeb34 please fill out the PR details section. it is currently using the template

Resolved

**Comment by Nanayeb34** [comment]
> @Nanayeb34 please fill out the PR details section. it is currently using the template

@wsedor resolved

**Review by wsedor — APPROVED** [review]

**Pull request merged**